### DISCUSSION TITLE
Stopped after homing at start of print

### DISCUSSION THREAD

**Discussion by Onejk1** [Jyers/Marlin#333] [Q&A] [answered]
so just installed BLtouch and gone for the 4.2.2 10x10 HS. i have complete a mesh ok and saved it, i have set the the z offset ok. set the x and y probe offset. Every time I go to printer its zero's x/y fine, moves to the center and probes the bed twice and the probe then stays retracted and solid red. it moves back up 10mm and then displays STOPPED on the screen. From Octoprint terminal it says stopped due to error.

Any ideas what i am doing wrong. been using your firmware on manual mesh no problems. BTW thank you for the excellent versions

**Comment by Onejk1** [comment, marked as answer]
also when doing the 100 mesh i'm sure it stopped after point 91

**Reply by Onejk1** [reply]
managed to get it to work, reset setting and then it would print however even though i had already set the z offset and it was still showing as the same value, it score the glass, so had to reset it. very strange behaviour. i thing its probably a good idea to hit reset setting after loading a new firmware

**Reply by Jyers** [reply]
Resetting after flashing is usually a good idea, also the 91/100 is due to there being points on the bed that the probe can not reach. These extra points will be inferred based on the probed points to form the mesh.
